inc/saiswitch.h

```diff
@@ -1460,6 +1460,15 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_PFC_TC_DLR_INTERVAL,
 
     /**
+     * @brief Get the list of supported protected object types.
+     *        See comment for SAI_NEXT_HOP_GROUP_MEMBER_ATTR_MONITORED_OBJECT for more details.
+     *
+     * @type sai_s32_list_t
```

Comment:
I've added this as you suggested, but then had to change the name of the attribute such that it ends with OBJECT_TYPE.  Is it not misleading if we're talking about  a list of object types?